src/dialogs/settingsdialog.ui

```diff
@@ -22,6 +22,12 @@
     <height>16777215</height>
    </size>
   </property>
+  <property name="font">
```

Comment:
do we need that?

Reply:
I used to it change the font , i manually changed the font which probably generated this block

Reply:
fonts will be decided by the operating system